integrations/docker/images/stage-3/chip-build-android/Dockerfile

```diff
@@ -40,11 +40,11 @@ RUN set -x \
 # Download and install android NDK
 RUN set -x \
     && cd /tmp \
-    && wget https://dl.google.com/android/repository/android-ndk-r23c-linux.zip \
+    && wget https://dl.google.com/android/repository/android-ndk-r28c-linux.zip \
     && mkdir -p /opt/android \
     && cd /opt/android \
-    && unzip /tmp/android-ndk-r23c-linux.zip \
-    && rm -f /tmp/android-ndk-r23c-linux.zip \
+    && unzip /tmp/android-ndk-r28c-linux.zip \
```

Comment:
These changes have to be in a separate PR with a version file update.

We always have to do updates in lockstep: Docker first, then changes to update the docker image.

Reply:
done, thanks